content/md/en/docs/main-docs/build/build-process.md

```diff
@@ -1,118 +1,155 @@
 ---
 title: Build process
-description: This article describes what happens under the hood when building and executing a Substrate node.
+description: Describes how a Substrate node is compiled into Rust and WebAssembly binaries and how the binaries are used to execute calls into the Substrate runtime.
 keywords:
 ---
 
-This article describes what happens under the hood when building and executing a Substrate node.
+In [Architecture](/main-docs/fundamentals/architecture), you learned that a Substrate node consists of two main environments—an outer node host environment and a runtime execution environment—and that these environments communicate with each other through runtime API calls and host function calls.
+In this section, you'll learn more about how the Substrate runtime is compiled into a standard Rust executable that is included as part of the node executable and into a WebAssembly (Wasm) binary that is stored on the blockchain.
```

Comment:
"rust executable" makes no sense, same as above. There is a native executable. Both Wasm and native executable come from "rust"